pkg/orchestration/wiring/wiringutil/knownshapes/asap_key.go

```diff
@@ -0,0 +1,43 @@
+package knownshapes
+
+import (
+	smith_v1 "github.com/atlassian/smith/pkg/apis/smith/v1"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin"
+)
+
+const (
+	ASAPKeyShapeName wiringplugin.ShapeName = "voyager.atl-paas.net/ASAPKeyShape"
+)
+
+// +k8s:deepcopy-gen=true
+// +k8s:deepcopy-gen:interfaces=github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin.Shape
+type ASAPKey struct {
+	wiringplugin.ShapeMeta `json:",inline"`
```

Comment:
Shouldn't ASAPKey shape output some data? (ASAP private key etc)? i.e. there should be additional field in the struct.
Shape without data seems useless...

Reply:
@scottgreenup asked @ychen-atlassian annd this is a no for this PR

Reply:
We may need to add data later. e.g. bucket urls.